combat-trainer.lic

```diff
@@ -2736,6 +2758,89 @@ class TrainerProcess
          'dumping some dirt', 'What were you referring')
   end
 
+  def pickbox(game_state, number='first')
```

Comment:
How much of this is a direct copy from `lockbox.lic`, `locksmithing.lic`, or `pick.lic`? I.e. if there is a bunch of overlapping/copied code, is it time to create a `common-locksmithing` or something?

Reply:
If anything, it would overlap with lockbox.lic and `def daily_lockbox` and `def consumable_lockbox` from locksmithing. Perhaps we could make `common-locksmithing`.

Reply:
Also, I did not take into acount worn trainers. I will need to cover in the `common-locksmithing` file.